lib/srv/db/server.go

```diff
@@ -592,6 +610,10 @@ func (s *Server) Start(ctx context.Context) (err error) {
 		s.cfg.OnHeartbeat(nil)
 	}
 
+	if s.cfg.CloudLabels != nil {
+		s.cfg.CloudLabels.Start(ctx)
```

Comment:
If there are multiple services in the same process (e.g. app agent, db agent, ssh agent, etc.), will cloud labels fetcher be started multiple times? Can we start it once in the service.go and pass already started instance to all services?